src/index.js

```diff
@@ -36,6 +36,11 @@ async function boot () {
       keyName: readinessConfig.dynamo.keyName,
       keyValue: readinessConfig.dynamo.keyValue
     })
+    const taggedPeers = await awsClient.dynamoGetItem({
```

Comment:
What happen if doesn't find the peers?

Reply:
If the query fails, the service won't start
If the taggedPeers are empty, nothing